modules/book.js

```diff
@@ -0,0 +1,195 @@
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default class Book {
+  constructor() {
+    this.awesomeBooks = document.querySelector('#awesome-books');
+    this.formSelector = document.querySelector('form');
+    this.titleSelector = document.getElementById('title');
+    this.authorSelector = document.getElementById('author');
+    this.allBtnSelector = this.awesomeBooks.querySelectorAll('.btn');
+
+    this.books = JSON.parse(localStorage.getItem('booksLocalStorage')) || [];
+    this.title = localStorage.getItem('currentTitle') || null;
+    this.author = localStorage.getItem('currentAuthor') || null;
+    this.listSelector = document.getElementById('list-navigation');
+    this.addSeclector = document.getElementById('add-navigation');
+    this.contactSeclector = document.getElementById('contact-navigation');
+    this.navigationSelector = document.getElementById('navigation');
+
+    this.addSection = document.getElementById('added-book');
+    this.formSection = document.getElementById('form-container');
+    this.contactSection = document.getElementById('contact');
+  }
+
+  /* Display an added book to the UI */
+  display() {
+    this.awesomeBooks.innerHTML += `
+  
+      <li id="${this.id}">
+      <h2>"${this.title}"  by   ${this.author}</h2>
+      <button class="btn button-shadow">remove</button>
+      </li>
+  
+      `;
+  }
+
+  /*
+Filter out a removed book from an array of books contianed in each list item
+- add new array of books (with one less book) to the local storage
+*/
+  remove(id) {
+    this.books = this.books.filter((book) => book.id !== id);
+    localStorage.setItem('booksLocalStorage', JSON.stringify(this.books));
+    this.removeBorderFromBooks();
+  }
+
```

Comment:
- [ ] As part of the requirements, please refactor your methods and functions into arrow functions 
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/37901469/199006655-1decc649-5960-48d3-a12c-988cf32a450f.png">